README.md

```diff
@@ -30,7 +30,17 @@ You may also optionally place the json settings directly into the `ENV_JSON` env
     username: root
     password: root
     ```
-1. As you are now logged in as `root`, there are no courses listed. Navigate to http://localhost:5001/courses/231768 or http://localhost:5001/courses/430174 to view a sample course as a superuser.
+1. As you are now logged in as `root`, there are no courses listed. Next 3 steps will help you view a sample course.
+1. Connect to MySQL database.
+    ```
+    Host: 127.0.0.1
+    Username: student_dashboard_user
+    Password: student_dashboard_pw
+    Database: student_dashboard
+    Port: 5306
+    ```
+1. Navigate to `course` table and select canvas_id `canvas_id`("SELECT canvas_id From course"), which will be used in the next step.
+1. Nagivate to http://localhost:5001/courses/{canvas_id} with the canvas_id you found. (For example, with SQL file myla_test_data_2019_10_16.sql loaded in, nagivate to http://localhost:5001/courses/235420 or http://localhost:5001/courses/362855 and you can view the course page as an admin.) 
```

Comment:
I feel like with the changes introduced in PR #761, we probably shouldn't be listing any hardcoded course values and just instruct people to use the admin. So 

```
1. As you are now logged in as `root`, there are no courses listed. To view the Django admin interface, navigate to http://localhost:5001/admin
1. Click on "Courses" under "Dashboard" to view a list of sample courses.
1. Click on the link in the "Course Link" column for the sample course you wish to view.
```

@jonespm, thoughts?

Reply:
@jpyan, thank you for trying to improve the README, but because this isn't really related to the Snackbar issue, let's remove these changes for now. I'm going to move my comment to issue #719, where it's more directly related and our team will address this separately.

Reply:
For sure. It was actually related to closed PR 811 https://github.com/tl-its-umich-edu/my-learning-analytics/pull/811. I guess I'm not familiar with github enough to separate my commits to different PR. I can just git revert those few commits related to README. Does that sound good to you? The change is already being merged into the master.

Reply:
Oh, I see. Thanks for filling me in on this. To avoid this in the future, you might create separate branches for your work and then submit PRs from those branches. So for instance, you could have submitted this PR from an `issue-806-fix-snackbar-duration` branch` and the other from an `update-readme` branch.

I think we'll be okay merging this, if the other commits are in there. Let me consider for a moment.

Reply:
Thanks a lot for future suggestion! Let me know if anything needed to be changed.